lib/appoptics_apm/frameworks/grape.rb

```diff
@@ -33,6 +33,7 @@ def run_with_appoptics(*args)
         else
           report_kvs[:Action] = args.empty? ? env['PATH_INFO'] : args[0]['PATH_INFO']
         end
+        report_kvs[:Backtrace] = AppOpticsAPM::API.backtrace if AppOpticsAPM::Config[:grape][:collect_backtraces]
```

Comment:
How would I find where AppOpticsAPM::Config is defined? I'm digging for how to look at namespaces and determine how that maps to the files.

Reply:
The file /Users/maiaengeli/workspace/librato/ruby-appoptics/lib/appoptics_apm/config.rb
defines `AppOpticsAPM::Config` and loads the settings from the config file.

  ```
    # There are 3 possible locations for the config file:
    # Rails default, ENV['APPOPTICS_APM_CONFIG_RUBY'], or the gem's default
    #
    # Hierarchie:
    # 1 - Rails default: config/initializers/appoptics_apm.rb
    #     (also loaded  by Rails, but we can't reliably determine if Rails is running)
    # 2 - ENV['APPOPTICS_APM_CONFIG_RUBY']
    # 3 - Gem default: <startup_dir>/appoptics_apm_config.rb
```

Reply:
usually the filename is the lowercase version of the Module/Class name it defines